dask_ml/model_selection/_incremental.py

```diff
@@ -476,26 +476,55 @@ def _get_history_results(self, results):
         * partial_fit_calls
         * model_id
         """
-        info, model, history = results
+        _, model, history = results
+        info = {}
         key = operator.itemgetter("model_id")
         hist2 = sorted(history, key=key)
-        return hist2
-
-    def _get_best(self, results, history_results):
-        # type: (Dict, Dict) -> Estimator
-        """Select the best estimator from the set of estimators."""
-        best_model_id = first(results.info)
-        key = operator.itemgetter("model_id")
-        best_index = -1
-        # history_results is sorted by (model_id, partial_fit_calls)
-        # best is the model_id with the highest partial fit calls
-        for k, v in itertools.groupby(history_results, key=key):
-            v = list(v)
-            best_index += len(v)
-            if k == best_model_id:
-                break
+        for h in hist2:
```

Comment:
Should this just be done by `_fit`?

Reply:
I've made the change which is makes `info` have the same data as `history`, but in a different format. This breaks a test:

https://github.com/dask/dask-ml/blob/b5b6954cbd8a4da89aa4da9612e970a0680290da/tests/model_selection/test_incremental.py#L181

The relevant bits of my diff are

``` diff
- assert len(info) == len(models) == 1
+ assert len(info) > len(models) == 1
+ assert set(models.keys()).issubset(set(info.keys()))
+
+ calls = {k: [h["partial_fit_calls"] for h in hist] for k, hist in info.items()}
+ for k, call in calls.items():
+     assert (np.diff(call) >= 1).all()

```

This change provides some ease-of-use and another test. Do we want to make this change?

Reply:
Can you post what `info` looks like now? Breaking tests is fine for now, if we think it's a better data format.